nixos/modules/misc/man-db.nix

```diff
@@ -0,0 +1,61 @@
+{ config, pkgs, lib, ... }:
+
+let
+  cfg = config.documentation.man.man-db;
+in
+
+{
+  options = {
+    documentation.man.man-db = {
+      enable = lib.mkEnableOption "man-db as the default man page viewer" // {
+        default = true;
```

Comment:
This default should be wired up to `config.documentation.man.enable` if possible.

Reply:
Very good suggestion, done!